src/video/sdl2_v.cpp

```diff
@@ -367,13 +367,18 @@ static uint ConvertSdlKeycodeIntoMy(SDL_Keycode kc)
 bool VideoDriver_SDL_Base::PollEvent()
 {
 	SDL_Event ev;
+	// workaround for old SDL libraries
+	bool is_SDL_2_0 = this->sdl_lib_major == 2 && this->sdl_lib_minor < 24;
```

Comment:
Could we use [SDL_VERSION_ATLEAST](https://wiki.libsdl.org/SDL2/SDL_VERSION_ATLEAST) instead?